src/platform/mock-local-home-platform.ts

```diff
@@ -100,10 +102,11 @@ export class MockLocalHomePlatform {
   }
 
   /**
-   * Forms an `ExecuteRequest` with the given commands and passes it to the fulfillment app.
+   * Forms an `ExecuteRequest` with the given commands.
+   * Passes it to the fulfillment app.
    * @param requestId  The request id to set in the `ExecuteRequest`
```

Comment:
Is the plan to put periods at the end of each line? This is one line without a period.

Reply:
Yes.  Done. I noticed the Local Home SDK has periods at the end of all JSDocs so am going to generally follow that rule.  Went through and added periods to all comments - should be fairly consistent now.